README.md

```diff
@@ -76,6 +76,9 @@ SummaryProcessor accepts the following context in addition to the Processor's co
 
 ```ruby
 {
+  greenmat: {
+    footnotes: false # Enable / Disable footnotes in document. (Default: true)
+  },
```

Comment:
This option should be described in the `Processor` section because it's also available there.